hack/trivy_scan.sh

```diff
@@ -0,0 +1,248 @@
+# !/usr/bin/env bash
```

Comment:
Do we want to migrate to python entirely for that script? What do you think @hansinikarunarathne